HubSpot.NET/Api/CustomObject/HubSpotCustomObjectApi.cs

```diff
@@ -242,4 +261,15 @@ public string UpdateObject<T>(T entity) where T : UpdateCustomObjectHubSpotModel
         
         return string.Empty;
     }
+
+    public T GetEquipmentHourMileage<T>(string schemaId, string entityId) where T : GetHubspotHoursMileageModel, new()
```

Comment:
It might be worth making this function more generic to `GetEquipment` and include some more properties on it (ex. make, model, year, category) so that we can use it for syncing those additional fields from Hubspot in the future.

Reply:
also the vin/serial number - it think it's just called `vin` in hubspot